src/components/header/_macro.njk

```diff
@@ -11,7 +11,30 @@
         {% set currentLanguageISOCode = currentLanguage.ISOCode %}
     {% endif %}
 
+    {# Unsupported browser banner #}
```

Comment:
I would pass the `currentLanguageISOCode` into the browser banner macro and then move this language logic inside it.